nodejs/week1/avg.js

```diff
@@ -0,0 +1,17 @@
+
+const arg = process.argv.slice(2);
```

Comment:
Great idea to assign these confusing values (`process.argv`) into variables with clear names!